client/src/components/SubmissionTabs/SubmissionTabs.tsx

```diff
@@ -46,18 +61,26 @@ function SubmissionTabs({ card }: cardProps): JSX.Element {
         variant="fullWidth"
         onChange={handleChange}
       >
-        <Tab label="Designs (30)" />
+        <Tab label={countImages()} />
```

Comment:
Image files is an array, so maybe just display the length of the array here?

Reply:
@rajivtitus Some submissions have an array of imagefiles. I am counting each of them and adding it to the length of the array of imagefiles of the other submissions. I will make changes to the countImages function by using length instead of map